### PR TITLE
Dev change

### PR DESCRIPTION
**Pull request by msamandi**
please review my request

### CODE REVIEW

File: .mvn/settings.xml [hunk 1]
```diff
@@ -32,6 +32,11 @@
       <username>${artifactory.username}</username>
```
Comment (johnsall) [inline .mvn/settings.xml:32]: Remove the artifactory creds

File: build/azDevOps/azure/azuredevops-vars.yml [hunk 2]
```diff
@@ -90,7 +90,7 @@ variables:
   - name: maven_package_version
     ${{ if eq( variables['Build.SourceBranchName'], 'main' ) }}:
       value: "${{ variables.version_major }}.${{ variables.version_minor }}.${{ variables.version_patch }}"
-    ${{ if ne( variables['Build.SourceBranchName'], 'main' ) }}:
+    ${{ if ne( variables['Build.SourceBranchName'], 'dev-change' ) }}:
```
Comment (johnsall) [inline build/azDevOps/azure/azuredevops-vars.yml:93]: Why this?

File: pom.xml [hunk 3]
```diff
@@ -6,11 +6,34 @@
     <groupId>com.amido.stacks.modules</groupId>
     <artifactId>stacks-modules-parent</artifactId>
     <version>1.0.0-SNAPSHOT</version>
-    <name>stacks-modules-parent</name>
+    <name>${project.groupId}:${project.artifactId}</name>
```
Comment (johnsall) [inline pom.xml:9]: Have you confirmed this concat has the same result as before?

**Review by johnsall — CHANGES_REQUESTED** [review]
Source branch needs to be named back to main from dev-change

File: build/azDevOps/azure/azuredevops-vars.yml (hunk 2 above)
Comment (msamandi) [reply build/azDevOps/azure/azuredevops-vars.yml:93]: it just a quick change to test a branch, i rolled back

File: pom.xml (hunk 3 above)
Comment (msamandi) [reply pom.xml:9]: this the best practise to name a project , the concat  is different as the name include the group id as well

File: build/azDevOps/azure/azuredevops-vars.yml (hunk 2 above)
Comment (johnsall) [reply build/azDevOps/azure/azuredevops-vars.yml:93]: I'll leave the comment here till then, so we won't forget

**Review by johnsall — APPROVED** [review]

**Comment by msamandi** [comment]
withdraw

**Pull request closed without merging**